pages/quiz/[question].js

```diff
@@ -239,25 +237,19 @@ const NavBar = styled.footer`
 const StyledQuizButton = styled.button`
   font-family: Georgia, 'Times New Roman', Times, serif;
   font-size: 42px;
-  padding-bottom: 9px;
+  padding-bottom: 7px;
```

Comment:
Again, for future reference: this is what's called a 'magic number'. At first glance, I'm not sure why you've chosen 7px specifically. You can use variables for spacing/etc, as well. This goes for all of your hardcoded units of measurement (most px widths, etc). It might seem counter-intuitive at first, but you could define a variable such as `main-padding: 1rem;` and call it with `var(--main-padding)`, so people can understand at first glance when the same variable shows up in a few different places what it does, and how it can be used. : )